app/src/main/java/org/thoughtcrime/securesms/ui/Util.kt

```diff
@@ -3,9 +3,18 @@ package org.thoughtcrime.securesms.ui
 import android.app.Activity
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.platform.ComposeView
+import androidx.fragment.app.Fragment
 
 fun Activity.setComposeContent(content: @Composable () -> Unit) {
     ComposeView(this)
-        .apply { setContent { AppTheme { content() } } }
+        .apply { setContent { SessionMaterialTheme { content() } } }
         .let(::setContentView)
 }
+
+fun Fragment.onCreateView(content: @Composable () -> Unit): ComposeView = ComposeView(requireContext()).apply {
```

Comment:
Could we maybe name this `setComposeContent` for consistency with the Activity function?

Reply:
that one calls `setContent` on the `Activity`, so that wouldn't be entirely accurate.

`setComposeContent` calls `setContent`
so `onCreateView` can call `onCreateComposeView` ?

Reply:
Well 'onCreateView' is a callback from the Fragment's lifecycle.
The method we're creating is an action, not a callback, right? It's actually creating a view as a compose content?

Reply:
k, my function just creates a themed `ComposeView` from a `Context` or... a `Fragment`, so could be `createThemedComposeView`